base/applications/rapps/appview.cpp

```diff
@@ -1299,7 +1299,7 @@ CAppsListView::AddApplication(CAppInfo *AppInfo, BOOL InitialCheckState)
             hIcon = ExtractIconW(hInst, szIconPath.GetString(), 0);
         }
 
-        if (!hIcon)
+        if (!hIcon || (hIcon == (HICON)(INT_PTR)1))
```

Comment:
@katahiromz Does using `(HICON)1` alone (as done here https://git.reactos.org/?p=reactos.git;a=blob;f=dll/win32/shell32/wine/shell32_main.c;hb=561ad3c1d6ab4d5f617e54e03a3e9f0feb46cc70#l894 ) trigger a build warning or error?

Reply:
It will cause a warning on x64.

Reply:
> It will cause a warning on x64.

I checked a few amd64 build logs, it would seem there is no such warning on existing similar code.
Could you be explicit about which warning on which build you got? Thanks.

Reply:
Fixed as `(HICON)1`.